tgui/packages/tgui/interfaces/Vending.tsx

```diff
@@ -1,32 +1,85 @@
-import { classes } from 'common/react';
+import { classes } from '../../common/react';
```

Comment:
Revert this, you are not supposed to escape package boundaries

Reply:
The code was refusing to build in tsx without this